### DISCUSSION TITLE
Disable goto percentage on desktop.

### DISCUSSION THREAD

**Discussion by BasicAcid** [koreader/koreader#10225] [Q&A] [answered]
Hello, 

I am using KoReader desktop on Linux.

When I press a number from the numeric keys, the behavior is to go to the n% of the document.

I would like to disable this feature, but I didn't find a mention to it in the docs.

Why this is bothering me: I am using a window manager (xmonad), and when I go to a workspace where a document is already open (using a combination like Meta+number), then the num key is also interpreted by koreader and it sends me to another page.

Note: I don't know if this is of any importance, but it might be linked:

I am using a french (azerty) keyboard, the behavior is a bit different regarding the numeric keys compared to qwerty, so here is an example of what is happenning:

& ->    10%
é ->     20%
" ->     30%
...

Thanks for your help.

**Comment by hius07** [comment, marked as answer]
https://github.com/koreader/koreader/issues/8992

**Reply by Frenzie** [reply]
That being said, I do have my doubts as to how useful these are to have there at all. Presumably it was felt to be useful on the original Kindle; is that still the case?
